crates/sui-move-build/src/lib.rs

```diff
@@ -233,6 +237,39 @@ pub fn build_from_resolution_graph(
 ) -> SuiResult<CompiledPackage> {
     let (published_at, dependency_ids) = gather_published_ids(&resolution_graph);
 
+    // collect bytecode dependencies as these are not returned as part of core
+    // `CompiledPackage`
```

Comment:
I feel like they should be? If this is an issue

Reply:
Though needing the full deserialized bytes feels maybe like a Sui specific thing? Unsure

Reply:
I'm not nearly as familiar with this codebase as any of you are, but I'm also unsure. Though here the only reason I've added these here is because of source verification (to avoid having to hackily load them somehow in the source verification implementation), which is a Sui specific thing.So perhaps it's justifiable keeping them here unless there's another reason to have them in core `CompiledPackage`?